src/app/src/components/ClaimFacilityStepper.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import Stepper from '@material-ui/core/Stepper';
+import Step from '@material-ui/core/Step';
+import StepLabel from '@material-ui/core/StepLabel';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+
+const claimFacilityStepperStyles = Object.freeze({
+    containerStyles: Object.freeze({
+        width: '100%',
+    }),
+    controlContainerStyles: Object.freeze({
+        width: '100%',
+        padding: '24px',
+    }),
+    buttonsContainerStyles: Object.freeze({
+        padding: '12px 0',
+    }),
+    buttonStyles: Object.freeze({
+        margin: '5px',
+    }),
+});
+
+const steps = Object.freeze([
+    Object.freeze({
+        name: 'Contact Information',
+    }),
+    Object.freeze({
+        name: 'Facility Information',
+    }),
+    Object.freeze({
+        name: 'Location Information',
+    }),
+    Object.freeze({
+        name: 'Finish Up',
+    }),
+]);
+
+export default function ClaimFacilityStepper() {
+    const [activeStep, setActiveStep] = useState(0);
+
+    const incrementActiveStep = () => setActiveStep(activeStep + 1);
+    const decrementActiveStep = () => setActiveStep(activeStep - 1);
```

Comment:
Reminder to me to clamp these values between 0 & `step.length`: https://lodash.com/docs/4.17.11#clamp